conf/paramsheet.yaml

```diff
@@ -12,6 +15,7 @@
   observations_id_col: sample
   observations_name_col: sample
 
+# rnaseq analysis with deseq2
```

Comment:
maybe include comments as "description" field in the YAML? It could be useful to have access to that from within the pipeline.